docs/data/material/components/rating/BasicRating.tsx

```diff
@@ -16,6 +16,13 @@ export default function BasicRating() {
           setValue(newValue);
         }}
       />
+      <Typography component="legend">Uncontrolled</Typography>
+      <Rating
+        name="simple-uncontrolled"
```

Comment:
I think you can set `defaultValue` for uncontrolled because other components also have a value.
```suggestion
        name="simple-uncontrolled"
        defaultValue={2}
```